build-system/scripts/setup_env

```diff
@@ -25,7 +25,7 @@ ROOT_PATH=$(git rev-parse --show-toplevel)
 BUILD_SYSTEM_PATH=$(cd "$(dirname "${BASH_SOURCE[0]}")/.." && pwd)
 PROJECT=$(cat $ROOT_PATH/PROJECT)
 COMMIT_MESSAGE=$(git log -n 1 --pretty=format:"%s" $COMMIT_HASH)
-PATH=$PATH:$BUILD_SYSTEM_PATH/scripts
+PATH=$PATH:$BUILD_SYSTEM_PATH/scripts:$BUILD_SYSTEM_PATH/s3-cache-scripts
```

Comment:
I still prefer this being isolated in each script, I don't love relying on BASH_ENV. Same for the aws creds, ideally they'd be exported just in bootstrap.sh as needed - otherwise rogue software can read them, which tbh I don't view as a big risk, but some people view it otherwise